pandas/core/generic.py

```diff
@@ -2312,6 +2312,13 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
         --------
         pandas.read_sql : read a DataFrame from a table
 
+        Notes
+        -----
+        Timezone aware datetime columns will be written as
+        ``Timestamp with timezone`` type with SQLAlchemy if supported by the
+        database. Otherwise, the datetimes will stored in the closest available
```

Comment:
stored -> be stored

I think "closest available type" is a bit unclear. I would just say that they will be stored as "timezone unaware timestamps local ..."